vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddChoiceBlockController.java

```diff
@@ -0,0 +1,32 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.model.IChoiceBlock;
+import edu.asu.diging.vspace.core.services.IContentBlockManager;
+
+@Controller
+public class AddChoiceBlockController {
+
+    @Autowired
+    private IContentBlockManager contentBlockManager;
+
+    @RequestMapping(value = "/staff/module/{moduleId}/slide/{id}/choicecontent", method = RequestMethod.POST)
+    public ResponseEntity<IChoiceBlock> addChoiceBlock(@PathVariable("id") String slideId,
+            @PathVariable("moduleId") String moduleId, @RequestParam("content") String choiceId,
```

Comment:
is it content or a choice id?

Reply:
Also, I think we said that the choice block would for now have all choices of the branching point.

Reply:
Currently, we are popping the screen showing all the available choices through which the user can select the one that should be added as content. 
should this option be removed and defaulted to link all the choices of branching point or user should be able to select the choices just that user should select all at a time from the branching points list?

Reply:
jsut default to all choices; we'll add features to only add specific ones later.